android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -213,15 +213,14 @@ private void configureRuntimeOptions(Client client, ReadableMap options) {
       }
 
       if (options.hasKey("endpoint")) {
-          String endpoint = options.getString("endpoint");
-          if (endpoint != null && endpoint.length() > 0)
-              client.setEndpoint(endpoint);
-      }
+          String notify = options.getString("endpoint");
 
-      if (options.hasKey("sessionsEndpoint")) {
-          String endpoint = options.getString("sessionsEndpoint");
-          if (endpoint != null && endpoint.length() > 0)
-              config.setSessionEndpoint(endpoint);
+          if (notify != null && notify.length() > 0) {
+              if (options.hasKey("sessionsEndpoint")) {
+                  String sessions = options.getString("sessionsEndpoint");
+                  config.setEndpoints(notify, sessions);
+              }
+          }
       }
```

Comment:
There should be a log message here if only `sessionsEndpoint` is set

Reply:
A warning is logged by bugsnag-android, if that's sufficient? https://github.com/bugsnag/bugsnag-android/blob/master/sdk/src/main/java/com/bugsnag/android/Configuration.java#L160

Reply:
In this case it would never get to the bugsnag-android warning since `setEndpoints` is only called if both are set.